src/routes/admin-view.js

```diff
@@ -0,0 +1,143 @@
+import Prism from 'prismjs'
+
+import {
+  getCustomsDeclaration as getCustomsDeclarations,
+  getImportPreNotification as getImportPreNotifications,
+  getResourceEvents
+} from '../services/trade-imports-api.js'
+import { paths, CACHE_CONTROL_NO_STORE } from './route-constants.js'
+import { searchPatterns } from '../services/search-patterns.js'
+import { getRawMessages } from '../services/trade-imports-processor.js'
+import { STATUS_CODES } from '../utils/api.js'
+
+const ADMIN_VIEW_TEMPLATE = 'admin-view'
+const VALID_TYPES = new Set(['all-events', 'all-messages', 'information'])
+
+const TYPE_HANDLERS = {
+  'all-events': getResourceEvents,
+  'all-messages': getRawMessages
+}
+
+const INFORMATION_HANDLERS = {
```

Comment:
given INFORMATION_HANDLERS and TYPE_HANDLERS are only used in the `fetchResults` function it makes sense for them to be scoped to that function and declared there